static/templates/bookend.hbs

```diff
@@ -1,20 +1,26 @@
 {{! Client-side Mustache template for rendering the trailing bookend.}}
 <div class="{{#if trailing}}trailing_bookend{{/if}} bookend sub-unsub-message">
     <span>
-        {{#if deactivated}}
-            {{t "This stream has been deactivated" }}
+        {{#if is_spectator}}
+            {{#tr}}
+                Browse <z-link>recent topics</z-link>.
+                {{#*inline "z-link"}}<a href="#recent_topics">{{> @partial-block}}</a>{{/inline}}
+            {{/tr}}
         {{else}}
```

Comment:
I think Handlebars supports an `else if` syntax, which would avoid a lot of this indentation?

Reply:
Oops, thanks!